### PR TITLE
Create storage methods for Master File, and restructured the files

### PR DESCRIPTION
**Pull request by ivanderjmw**
Add storage methods for master files.

### CODE REVIEW

**Comment by felixhalim** [comment]
It doesn't pass Gradle check

File: src/main/java/seedu/trippie/Parser.java [hunk 1]
```diff
@@ -1,16 +1,6 @@
...
-import seedu.trippie.command.Command;
-import seedu.trippie.command.ExitCommand;
-import seedu.trippie.command.DeleteExpenditureCommand;
-import seedu.trippie.command.ListExpenseCommand;
-import seedu.trippie.command.BudgetCommand;
-import seedu.trippie.command.DeletePlaceCommand;
-import seedu.trippie.command.HelpCommand;
-import seedu.trippie.command.ListPlacesCommand;
-import seedu.trippie.command.SearchCommand;
+import seedu.trippie.command.*;
```
Comment (felixhalim) [inline src/main/java/seedu/trippie/Parser.java:3]: It is preferred to mention explicitly which packages are imported

File: src/main/java/seedu/trippie/Ui.java [hunk 2]
```diff
@@ -35,5 +35,9 @@ public String readCommand() {
         return in.nextLine();
     }
 
+    public String getLine() {
```
Comment (felixhalim) [inline src/main/java/seedu/trippie/Ui.java:38]: What is the difference between this and line 33?

**Review by felixhalim — COMMENTED** [review]
Great Job, Ivander! However, I have doubts on some parts. Please read my comment accordingly :D

File: src/main/java/seedu/trippie/Ui.java (hunk 2 above)
Comment (ivanderjmw) [reply src/main/java/seedu/trippie/Ui.java:38]: Unlike read command, this line is used without printing any symbols before the user input.

File: src/main/java/seedu/trippie/Parser.java [hunk 3]
```diff
@@ -1,30 +1,30 @@
...
             } else if (userInput.equals("list /e")) {
                 return new ListExpenseCommand();
-            } else if (userInput.startsWith("budget ")) {
+            } else if (userInput.startsWith("budget")) {
                 return new BudgetCommand(userInput);
-            } else if (userInput.equals("spending")) {
-                return new DisplayTotalExpenditureCommand();
             } else if (userInput.startsWith("add ")) {
                 return new AddPlaceCommand(userInput);
             } else if (userInput.startsWith("delete /p ")) {
```
Comment (ShawnTanzc) [inline src/main/java/seedu/trippie/Parser.java]: do we need to remove spaces for this too?

File: src/main/java/seedu/trippie/command/AddExpenseCommand.java [hunk 4]
```diff
@@ -5,39 +5,58 @@
...
         }
-        return expenseCost;
+        return Float.parseFloat(expenseCost);
+
     }
 
-    public String extractDayBought(String userInput) {
+    public int extractDayBought(String userInput) {
         String onlyDay = userInput.split(" /d ")[1];
-        return onlyDay.replaceAll("[^0-9]","").trim();
```
Comment (ShawnTanzc) [inline src/main/java/seedu/trippie/command/AddExpenseCommand.java]: I did this so it gives users the flexibility or options to type in "2" or "Day 2". so I dun think we should replace it?

File: src/main/java/seedu/trippie/Parser.java (hunk 3 above)
Comment (ivanderjmw) [reply src/main/java/seedu/trippie/Parser.java]: Umm.. did I remove the lines by accident?

File: src/main/java/seedu/trippie/command/AddExpenseCommand.java (hunk 4 above)
Comment (ivanderjmw) [reply src/main/java/seedu/trippie/command/AddExpenseCommand.java]: shawn I think there are some of your changes I accidentally deleted. It's the automerge..

File: src/main/java/seedu/trippie/Parser.java (hunk 1 above)
Comment (ivanderjmw) [reply src/main/java/seedu/trippie/Parser.java]: okk

File: src/main/java/seedu/trippie/command/AddExpenseCommand.java (hunk 4 above)
Comment (ivanderjmw) [reply src/main/java/seedu/trippie/command/AddExpenseCommand.java]: No, this was changed in this commit. But we could think of this feature again. [Link to commit.](https://github.com/AY2021S1-CS2113T-W11-2/tp/commit/a604d8334a4b5a00dc9181c49b5a63de6bee7a72)

**Pull request merged**